### ISSUE TITLE
Auto-run on Windows Start & Minimize to System Tray

### ISSUE THREAD

**Issue by Antzelino** [falkTX/Carla#1203]
I'd like to keep carla running on the background while im using it to filter white noise from my microphone, but it takes up space on my taskbar and i want to automate the process of starting it up, opening my project which has the settings i want for my microphone and minimize it to system tray.

It would be ideal to have an option in the settings to Run on Windows Startup, which would run Carla minimized in the System Tray with the last opened project active (so I don't have to expand it from System Tray to open the project and minimize it again).

If there is an easy & fast fix for this right now, I'd be glad to hear it and if it can be added in Carla, I would be super happy.

**Comment by Antzelino** [comment]
I found a way to automate the "Run on Windows Startup" and "Hide to System Tray" with a script, but I don't know how to run Carla with a specific project opened. Is there a way to do that?

**Comment by falkTX** [comment]
Yes, if you extract the Carla.exe (it is actually an executable zip-file).
The real carla.exe inside should handle being passed an argument as project file

**Comment by Antzelino** [comment]
It works! Thanks!
